packages/ui/src/components/Visualization/Custom/ContextMenu/item-interaction-helper.ts

```diff
@@ -63,3 +79,98 @@ export const processOnCopyAddon = (
 
   return processedContent;
 };
+
+const computeRelativePath = (parentPath: string, childPath: string): string | undefined => {
+  if (!childPath.startsWith(parentPath)) return;
+
+  const relativePath = childPath.substring(parentPath.length + 1);
+  return relativePath.replace(/\.(\d+)/g, '[$1]');
```

Comment:
Can we define the RegExp outside of the module so we don't need to recreate it on every execution? Also, we would need to reset it after each use with:
```ts
ID_REGEXP.lastIndex = 0;
```